src/db.js

```
@@ -1,34 +1,17 @@
-'use strict';
-
 const { Sequelize } = require('sequelize');
-const utils = require('util');
-
-// Needed for testing purposes, do not remove
-require('dotenv').config();
-global.TextEncoder = utils.TextEncoder;
-
-const {
-  POSTGRES_HOST,
-  POSTGRES_PORT,
-  POSTGRES_USER,
-  POSTGRES_PASSWORD,
-  POSTGRES_DB,
-} = process.env;
 
-/*
-  All credentials setted to default values (exsept password - it is exapmle)
-  replace if needed with your own
-*/
-
-const sequelize = new Sequelize({
-  database: POSTGRES_DB || 'postgres',
-  username: POSTGRES_USER || 'postgres',
-  host: POSTGRES_HOST || 'localhost',
-  dialect: 'postgres',
-  port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+const sequelize = new Sequelize('accounting', 'postgres', 'postgres', {
+    host: 'localhost',
+    dialect: 'postgres',
+    logging: false, // wyłączenie console.log w Sequelize
 });
 
-module.exports = {
-  sequelize,
-};
+async function testConnection() {
+    try {
+        await sequelize.authenticate();
+    } catch (error) {
+        console.error('Unable to connect to the database:', error);
```

`testConnection` currently catches errors and only logs them, which hides failures from callers. For automated tests and startup code you should surface the error (either rethrow after logging or return/throw an explicit error). For example:

```javascript
async function testConnection() {
  try {
    await sequelize.authenticate();
    return true;
  } catch (error) {
    console.error('Unable to connect to the database:', error);
    throw error; // <-- allow callers/tests to detect failure
  }
}
```

This prevents silent failures and makes CI/tests fail fast when DB is down.